### ISSUE TITLE
Glanthas the Ancient's Quest Markers are Very Small

### ISSUE THREAD

**Issue by Aubreykun** [slowtorta/turtlewow-bug-tracker#6412]
**Please, provide a clear description what the bug is:**

Despite being a huge Nordanaar-residing Ancient, the quest markers above Glanthas the Ancient's head are very small - at certain angles and distances it's barely visible behind his name. This is when he's involved in the quests "Runestone Reactivation" and "The Runestone Scepter" in Hyjal
See image:
![snapshot1014](https://github.com/slowtorta/turtlewow-bug-tracker/assets/150863259/c132c1df-a445-4557-9306-bd19397c8ee3)

**Steps to reproduce the behavior with as much detail as possible:**

1. Accept "Runestone Reactivation" (Obtained from Buthok Cloudhorn further down hyjal, but he is not impacted)
2. Go to talk to Glanthas.
3. See that the marker above his head is tiny.
4. Turn it in.
5. See that the subsequent markers are also tiny.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=61528
Quest: https://database.turtle-wow.org/?quest=40989
Quest: https://database.turtle-wow.org/?quest=40990

**Expected behavior. Describe how it should work:**

I believe other large NPCs have their question mark scaled to fit their sizes, if I'm not mistaken.

**Comment by twowTurtReynolds** [comment]
You are correct that other large NPCs have a large marker. I believe a serverside size attribute for the NPC governs things like the question mark and also the "you are targeting them" circle at their feet.
